spec/requests/create_dro_spec.rb

```diff
@@ -196,7 +201,8 @@
       expect(response).to have_http_status(:created)
       expect(IngestJob).to have_received(:perform_later).with(model_params: expected_model_params,
                                                               background_job_result: instance_of(BackgroundJobResult),
-                                                              signed_ids: signed_ids,
+                                                              signed_ids: {},
```

Comment:
Wondering the same about `signed_ids: {}` defaulting in the job's `#perform` method.